ui_framework/doc_site/src/views/color_picker/color_picker_clear.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { KuiColorPicker } from '../../../../components/index';
+
+export class ColorPickerLabelAndClear extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      color: null
+    };
+  }
+
+  handleChange = (value) => {
+    this.setState({ color: value });
+  };
+
+  render() {
+    return (
+      <KuiColorPicker
+        onChange={ this.handleChange }
+        color={ this.state.color }
+        label="Background Color"
+        showClearLink={ true }
+        showColorLabel={ false }
+      />
+    );
```

Comment:
Nice!  I tried to think of a way to do this but had trouble vertically aligning them.  I'm going to keep the example in the uiFramework but will use this method instead of putting the components inside the color picker itself.